packages/meteor-accounts-saml/saml_client.js

```diff
@@ -65,12 +67,12 @@ Meteor.loginWithSaml = function (options, callback) {
 	});
 };
 
-Meteor.logoutWithSaml = function (options, callback) {
-	//Accounts.saml.idpInitiatedSLO(options, callback); 
-	Meteor.call("samlLogout", options.provider, function (err, result) {
-		console.log("LOC " + result);
+Meteor.logoutWithSaml = function (options/*, callback*/) {
+	//Accounts.saml.idpInitiatedSLO(options, callback);
+	Meteor.call('samlLogout', options.provider, function (err, result) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["Meteor" is not defined.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2313371377/issues/source?bid=3199634&fileBranchId=3199636#l72)